source/playbooks/get-started-with-playbooks.rst

```diff
@@ -0,0 +1,55 @@
+Get started with Mattermost Playbooks
+=====================================
+
+Access Mattermost Playbooks by selecting **Playbooks** from the **Product list** |product-list| icon.
+
+.. |product-list| image:: ../images/products_E82F.svg
+  :height: 24px
+  :width: 24px
+  :alt: Navigate between Channels, Playbooks, and Boards using the Product list icon.
+
+Playbooks are workflows that can be repeated and refined over time. They can be applied to a range of situations and give stakeholders visibility into progress, process, and metrics. Playbooks are configurable and editable. You can add tasks to checklists, include automation behaviors, and provide insights into your processes in the form of metrics.
+
+When you get started with Playbooks, it helps to have a repeatable process in mind up front. As a working example, we'll refer to an incident response playbook and discuss its moving parts.
+
+Visit the `Mattermost Playbooks documentation <https://docs.mattermost.com/playbooks/setting-up-playbooks.html>`_ to learn more.
+
+Choose a template
+-----------------
+
+The first step is to choose the right template for your use case. There are pre-configured templates for specific scenarios. The checklists, actions, status updates, and retrospective settings for these templates may already be filled in and, where appropriate, enabled. You can always edit and adjust these settings - they're there to guide you - removing them doesn't negatively affect the playbook run.
+
+.. tip:: 
+
+    If you're unsure which option to choose, take a look at the **Learn how to use playbooks** template. This template breaks down the components of a playbook and you can also start a test run to see how everything fits together. If you're taking this option, you can stop reading here and enjoy the test run. You can also choose a blank template and start from scratch - this is a good option if your use case is unique.
+
+In the incident response template, the template contains items that are relevant to incident resolution. These are general items, to help you get started.
```

Comment:
The first sentence here reads a little odd to me but 0/5. 

Also, there's possibly an extra comma in the second sentence?
`These are general items to help you get started.`